translations/en-us.yaml

```diff
@@ -141,18 +141,22 @@ inbox:
     time: Time
     location: Location
     meeting-of: Meeting of
+    installation-meeting: Installation meeting
```

Comment:
I also heard 'Inauguration meeting', which seems to fit better as a translation for me. I could be wrong on the exact dutch meaning though.

Reply:
Good point, I updated it.